---
layout: post
title: CNS discovery methods upon multiple grass species
subtitle: STAG-CNS: An Order-Aware Conserved Noncoding Sequences Discovery Tool for Arbitrary Numbers of Species
---

This is my very first author paper published in my academic life. From few knowledge about coding and python to get much practice from this project, many thanks to my major advisor Dr. James Schnable. I started on the way of learning computational biology and inspiring new ideas on projects.

[**RNA-Seq Based Analysis of Population Structure within the Maize Inbred B73**](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0157942) was inspired from the idea that contaminations are boardly existed in diverse independent research groups around the world. Different from genomic data, as a reference variety, they used B73 a lot on RNA-seq data generation, as a common control line. This paper investigated 27 B73 public samples over the world and called SNPs to analyze population structure variation. Interestingly, samples from China seems further than those from USA to the reference genome. No worry, they just exist in few genes, expression altas did not show obvious bias in these variations. But different genomic blocks exist along genome sequences, which could be used to explain any potential blocks observed in other people's studies. 

<p align="center">
  <img width="500" height="400" src="https://i.imgur.com/SKIOH8U.png">
</p>
